inc/shortcodes/class-image-comparision.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Shortcake_Bakery\Shortcodes;
+
+class Image_Comparision extends Shortcode {
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => 'Image Comparison',
+			'listItemImage'  => 'dashicons-format-gallery',
+			'attrs'          => array(
+				array(
+					'label'  => 'Left Image',
+					'attr'   => 'left',
+					'type'   => 'attachment',
+					'libraryType' => array( 'image' ),
+					'addButton'   => 'Select Image',
+					'frameTitle'  => 'Select Image',
+					),
+				array(
+					'label'  => 'Right Image',
+					'attr'   => 'right',
+					'type'   => 'attachment',
+					'libraryType' => array( 'image' ),
+					'addButton'   => 'Select Image',
+					'frameTitle'  => 'Select Image',
+					),
+				array(
+					'label'  => 'Slider Start Position',
+					'attr'   => 'position',
+					'type'   => 'select',
+					'options' => array(
+						'center' => 'Center',
+						'mostlyleft' => 'Mostly Left',
+						'mostlyright' => 'Mostly Right',
+					),
+				),
+			),
+		);
+	}
+
+	public static function setup_actions() {
+		add_action( 'wp_enqueue_scripts', 'Shortcake_Bakery\Shortcodes\Image_Comparision::action_init_register_scripts' );
+		add_action( 'shortcode_ui_after_do_shortcode', function( $shortcode ) {
+			if ( false !== stripos( $shortcode, 'image-comparision' ) ) {
```

Comment:
Should this check be `if ( 0 === stripos( $shortcode, '[image-comparision' ) ) {` to be more precise?